src/services/fare.js

```diff
@@ -0,0 +1,196 @@
+const axios = require("axios");
+
+const { naverCloudApiId, naverCloudApiKey } = require("../../loadenv");
+const { taxiFareModel, locationModel } = require("../modules/stores/mongo");
+const { scaledTime } = require("../modules/fare");
+const logger = require("../modules/logger");
+
+// Naver Cloud Platform Maps Directions 5 API Keys
+const naverCloudApi = {
+  "X-NCP-APIGW-API-KEY-ID": naverCloudApiId,
+  "X-NCP-APIGW-API-KEY": naverCloudApiKey,
+};
+const naverCloudApiCall =
+  "https://naveropenapi.apigw.ntruss.com/map-direction/v1/driving?start=";
+
+/**
+ * 주어진 from, to, time에 대한 택시 요금을 반환합니다.
+ * @summary 카이스트 본원 <-> 대전역의 경로를 제외한 다른 경로의 경우, 1주일 전 매일 18:00시의 택시 요금을 반환합니다.
+ * @summary 카이스트 본원 <-> 대전역의 경우, cron으로 1주일 전 미리 캐싱해놓은 데이터를 기반으로 주어진 시간에 대한 택시 요금을 반환합니다. 만일, 해당 데이터가 존재하지 않을 경우에는 직접 호출해 보여줍니다.
+ * @param {Request} req - 파라미터로 from, to, time을 받습니다.
+ *  - @param {mongoose.Schema.Types.ObjectId} from - 출발지
+ *  - @param {mongoose.Schema.Types.ObjectId} to - 도착지
+ *  - @param {Date} time - 출발 시간 (ISO 8601)
+ */
+const getTaxiFare = async (req, res) => {
+  try {
+    if (
+      naverCloudApi["X-NCP-APIGW-API-KEY"] === null ||
+      naverCloudApi["X-NCP-APIGW-API-KEY-ID"] === null
+    ) {
+      logger.log(
+        "Naver Cloud API가 존재하지 않습니다.택시 비용 관련 기능을 사용할 수 없습니다."
+      );
+      res
+        .status(503)
+        .json({ error: "fare/getTaxiFare: Naver Cloud API not found" });
+      return;
+    }
+    const from = await locationModel
+      .findOne({
+        _id: { $eq: req.query.from },
+      })
+      .clone();
+    const to = await locationModel
+      .findOne({ _id: { $eq: req.query.to } })
+      .clone();
+    const sTime = scaledTime(new Date(req.query.time));
+
+    if (!from || !to) {
+      res.status(400).json({ error: "fare/getTaxiFare: Wrong location" });
+      return;
+    }
+
+    // 카이스트 본원 <-> 대전역
+    if (
+      (from.koName === "카이스트 본원" && to.koName === "대전역") ||
+      (from.koName === "대전역" && to.koName === "카이스트 본원")
+    ) {
+      const taxiFare = await taxiFareModel
+        .findOne(
+          {
+            from: from._id,
+            to: to._id,
+            time: sTime,
+          },
+          function (err, docs) {
```

Comment:
Taxi는 ES6 문법을 지향하기 때문에 화살표 방식으로 함수를 작성해주시면 좋습니다. 
```suggestion
          (err, docs) => {
```

관련 노션 문서 공유해드립니다.
https://www.notion.so/sparcs/ES6-3d8bbfc93f0c4630af19268ce5e8df77